packages/template-retail-react-app/app/pages/account/order-detail.jsx

```diff
@@ -233,48 +267,139 @@ const AccountOrderDetail = () => {
+                                                        <FormattedMessage
+                                                            defaultMessage="Pick up from Store {storeId}"
+                                                            id="account_order_detail.label.pickup_from_store"
+                                                            values={{
+                                                                storeId: shipment.c_fromStoreId
+                                                            }}
+                                                        />
+                                                    </Text>
+                                                )}
+                                            </Box>
+                                        </Stack>
+                                    )
+                                })}
 
-                                                    part_shipped: formatMessage({
-                                                        defaultMessage: 'Partially shipped',
-                                                        id: 'account_order_detail.shipping_status.part_shipped'
-                                                    }),
-                                                    shipped: formatMessage({
-                                                        defaultMessage: 'Shipped',
-                                                        id: 'account_order_detail.shipping_status.shipped'
-                                                    })
-                                                }[shippingStatus]
-                                            }
-                                        </Text>
-                                        <Text fontSize="sm">{shippingMethod.name}</Text>
-                                        <Text fontSize="sm">
-                                            <FormattedMessage
-                                                defaultMessage="Tracking Number"
-                                                id="account_order_detail.label.tracking_number"
-                                            />
-                                            :{' '}
-                                            {trackingNumber ||
-                                                formatMessage({
-                                                    defaultMessage: 'Pending',
-                                                    id: 'account_order_detail.label.pending_tracking_number'
-                                                })}
-                                        </Text>
-                                    </Box>
-                                </Stack>
+                                {/* Delivery Shipments */}
+                                {deliveryShipments.map((shipment, index) => (
+                                    <React.Fragment key={`delivery-${index}`}>
```

Comment:
the React.Fragment will not result in a DOM node at all, so the key prop here is useless. Why not use the Box component and pass the key there?